apps/aechannel/src/aesc_offchain_update.erl

```diff
@@ -1,6 +1,7 @@
 -module(aesc_offchain_update).
 
--define(UPDATE_VSN, 1).
+-define(UPDATE_VSN, 2).
```

Comment:
This makes [serialization](https://github.com/aeternity/aeternity/blob/2db747d7e378d4bf5235e532edc5a75ae150f0ed/apps/aechannel/src/aesc_offchain_update.erl#L265) to use v2 off-chain serialization. This is good but being not backwards compatible regarding the off-chain protocol communication, I expect [some system tests](https://github.com/aeternity/aeternity/blob/master/system_test/common/aest_channels_SUITE.erl#L16-L19) to fail